images/linux/Ubuntu1604-README.md

```diff
@@ -44,7 +44,7 @@
 ### Project Management
 - Ant 1.9.6
 - Gradle 7.0
-- Maven 3.6.3
+- Maven 3.8.1
```

Comment:
Please don't update autogenerated document files - *.md

Reply:
Done, I was not aware those were auto generated :+1: